test/lcmtypes/lcmtesttypes.jl

```diff
@@ -0,0 +1,136 @@
+module lcmtesttypes
+
+export lcm_test_type_1, lcm_test_type_2, lcm_test_type_3
+export hard_coded_example
+
+using LCMCore, StaticArrays
+
+mutable struct lcm_test_type_1 <: LCMType
+    a::Int16
+    blength::Int32
+    b::Vector{Int64}
+    c::SVector{3, Int32}
+end
+
+@lcmtypesetup(lcm_test_type_1,
```

Comment:
Does the fingerprint calculation for these types still get tested?

Reply:
Against the byte blobs in `test/lcmtypes`, yeah. Also here: https://github.com/JuliaRobotics/LCMCore.jl/pull/26/files#diff-99753e2a904c402cc41316398bf39d60R55.

Reply:
Ok, perfect.